### DISCUSSION TITLE
Change of Linux Distro

### DISCUSSION THREAD

**Discussion by dmandinec** [ish-app/ish#2315] [General]
Hello, this may be stupid question, but can i change shipped distro (Alpine) with some other distro, like fedora or debian, or other
Thank you very much in advance for answers.

Daniel.

**Comment by auser1337** [comment]
there is iSH-AOK, has support for Alpine, Debian 10, and Devuan 4

**Reply by dmandinec** [reply]
#I should mention that i use it on iphone from app store, is there any way to get like archive of that?

**Reply by auser1337** [reply]
what do you mean archive? do you mean how you install it? if so, there's a testflight for it [here](https://testflight.apple.com/join/X1flyiqE)
